1st-gen/tools/styles/README.md

```diff
@@ -1,124 +1,241 @@
+- **High contrast mode support** - Ensures text remains readable when Windows High Contrast Mode is enabled
+
+#### CSS imports
+
+For stylesheet use, import the complete typography system, with tokens:
+
+```css
+@import '@spectrum-web-components/styles/tokens-v2/dark-vars.css'; /* import color tokens */
+@import '@spectrum-web-components/styles/tokens-v2/large-vars.css'; /* import scale tokens */
+@import '@spectrum-web-components/styles/tokens-v2/global-vars.css'; /* import global tokens */
+@import '@spectrum-web-components/styles/typography.css';
 ```
-@import '@spectrum-web-components/styles/scale-large.css';
-```
 
-This file provides only the variables needed to power the "Large" scale system specification.
+#### TypeScript/JavaScript exports for Lit components
+
+For use in Lit-based components, you can import typography styles as JavaScript modules. Import the complete system or individual components depending on your needs:
+
+##### Import everything
 
-## Typography
+```js
+// tokens also need to be imported to define CSS custom properties
+import typographyStyles from '@spectrum-web-components/styles/typography.js';
 
+static styles = [typographyStyles];
 ```
-@import '@spectrum-web-components/styles/typography.css';
+
+##### Import only what you need
+
+For smaller bundle sizes, import individual components:
+
+```js
+import headingStyles from '@spectrum-web-components/styles/heading.js';
+
+static styles = [headingStyles];
 ```
 
-This file provides a `lit-html` compliant version of the [Spectrum Typography classes](https://opensource.adobe.com/spectrum-css/?path=/docs/components-typography--docs).
+##### Available JavaScript exports
+
+- **`typography.js`** - Complete typography system with all styles (heading, body, detail, code)
+- **`body.js`** - Body text styles only (includes base + lang + body)
+- **`heading.js`** - Heading styles only (includes base + lang + heading)
+- **`detail.js`** - Detail/label styles only (includes base + lang + detail)
+- **`code.js`** - Code styles only (includes base + lang + code)
+
+Each individual export (`body.js`, `heading.js`, etc.) includes the necessary base and language support styles, so you don't need to import them separately. Import `typography.js` for the complete system, or import individual exports to reduce bundle size.
+
+### Spectrum Vars tokens (deprecated)
+
+This package includes some deprecated files that use the older Spectrum Vars token naming convention, including combined theme files (e.g., `all-medium-dark.css`) and individual theme/scale files (e.g., `theme-light.css`, `scale-medium.css`).
+
+**Recommended alternatives:**
+
+- Use `<sp-theme>` for automatic theme management (recommended)
+- Import Spectrum Core tokens directly for granular control (see "Design tokens" section above)
+- See the [Spectrum Core tokens migration guide](../theme/core-tokens.md) for help migrating from Spectrum Vars to Spectrum Core tokens
```

Comment:
I don't think this link works. I think this should fix it: 

`tools/core-tokens/`